src/assets/scss/main.css

```diff
@@ -117,4 +117,8 @@
 .Terms p span {
   font-weight: 600;
   color: #242424;
+}
+
+.Family-Details {
+  width: 393px;
```

Comment:
Are fixed widths really necessary. Make sure these don't cause issues in smaller screens.